deeprank/domain/forcefield/__init__.py

```diff
@@ -44,6 +45,8 @@ def _find_matching_residue_class(self, residue):
 
     def get_vanderwaals_parameters(self, atom):
         type_ = self._get_type(atom)
+        if type_ is None:
+            return VanderwaalsParam(0.0, 0.0, 0.0, 0.0)
```

Comment:
Will these zeroes be added as features for a none type atom?

Reply:
yes they will. The calculated vanderwaals potential will be zero.